recipes/encebollado.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../recipes/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+    <title>Encebolladito</title>
+</head>
+<body>
+    <div class="principal">
+        <h1>Encebollado</h1>
+    <img src="../recipes/encebo.png" alt="Encebollado">
+    <br>
+    <h2>Así debe lucir el delicioso Encebollado</h2>
+    <p>Para preparar un delicioso Encebollado como el que vemos en la imagen debemos realizar lo siguiente</p>
+    </div>
```

Comment:
Nice job but here you can use semantic tags and use indentation

Reply:
[OPTIONAL] Great job on highlighting the student's effort :+1: . It is also important to let the student know why you are requesting a change and put the student on the correct path. In this way, we can make sure that the `assignee` fully understands what is being asked and they can proceed with the requested change. So please kindly take these factors into consideration while giving your next review :pray: